excel-export/src/helper.ts

```diff
@@ -0,0 +1,36 @@
+import strings from "./localizeStrings";
+
+export const statusMapping = (status) => {
+  switch (status) {
+    case "closed":
+      return strings.common.closed;
+    case "open":
+      return strings.common.open;
+    default:
+      return "unknown";
+  }
+};
+
+export const workflowItemTypeMapping = (type) => {
+  switch (type) {
+    case "general":
+      return strings.common.general;
+    case "restricted":
+      return strings.common.restricted;
+    default:
+      return "";
+  }
+};
+
+export const amountTypesMapping = (amountType) => {
+  switch (amountType) {
+    case "N/A":
+      return strings.workflowitem.na;
+    case "allocated":
+      return strings.workflowitem.allocated;
+    case "disbursed":
+      return strings.workflowitem.disbursed;
+    default:
+      break;
+  }
+};
```

Comment:
Proof if the right translations are used after changing the language. A problem occured in the frontend, when the string is allocated outside a React component which leads to a wrong translations
see: https://github.com/openkfw/TruBudget/pull/652